common/config/rush/common-versions.json

```diff
@@ -15,7 +15,8 @@
      * When someone asks for "^1.0.0" make sure they get "1.2.3" when working in this repo,
      * instead of the latest version.
      */
-    // "some-library": "1.2.3"
+    "prettier": "^1.16.4",
```

Comment:
I think this is abusing the `preferredVersions` feature to install these packages, and we should try to find a better way to install them.

Reply:
How is that abusing? I was actually following the `rush` guidelines itself: https://rushjs.io/pages/maintainer/git_hooks/

Reply:
I think you meant that `preferredVersions` typical use case is different. However, I do not think this is abusing because this behavior is exactly what we want in this case, having those two packages available as direct dependencies of the `common` package. Otherwise, we will have to add those two as dev deps to all packages and have the scripts inserted in all the `package.json` files. The downside to this latter approach is obviously the spamming and exposing scripts not intended to be run manually ever.

Reply:
I wasn't aware of the guidance at https://rushjs.io/pages/maintainer/git_hooks/.  If `rush` is recommending this approach it's fine with me.